docs/README.md

```diff
@@ -68,14 +68,73 @@ macOS (OS X) development environment.
 # User documentation
 
 * [Getting started (How-To with examples)](usage.md)
+
+## C/C++ Analysis
 * [Analyzer User guide](analyzer/user_guide.md)
-* [Webserver User guide](web/user_guide.md)
-* [Requiring credentials to view analysis results](web/authentication.md)
-* [Overview about connecting multiple analysis run databases](web/products.md)
-* [Permission management](web/permissions.md)
-* [Usage of PostgreSQL database](web/postgresql_setup.md)
-* [How to deal with false positives](analyzer/false_positives.md)
-* [Store Cppcheck analyzer reports](/tools/report-converter/README.md#cppcheck)
+* [Avoiding or suppressing false positives](analyzer/false_positives.md)
+* [Checker and Static Analyzer configuration](analyzer/checker_and_analyzer_configuration.md)
+* [GCC incompatibilities](analyzer/gcc_incompatibilities.md)
+* [Suppressing false positives](analyzer/user_guide.md#source-code-comments)
+
+## Web based report management
+* [Webserver User Guide](web/user_guide.md)
+* [Command line and WEB UI Feature overview](feature_comparison.md)
+
+ Security configuration 
```

Comment:
You can use sublist if you write one star (`*`) at the beginning of the line and indent the line with 2 spaces.
For example:
```md
* Item
  * Sub Item
```
It will generate the following:
* Item
  * Sub Item

I recommend to use it to format this list.

Reply:
I think the current paragraphing looks nice too. I like it even better.
Take a look at it here:
https://github.com/dkrupp/codechecker/tree/doc_fixes

Reply:
Yes, I already checked it but for me it's a little bit unstructured. 
![list](https://user-images.githubusercontent.com/6695818/81833295-d2c7a900-953f-11ea-8465-530cccd04822.png)

For example the above screenshot looks for me like we have 2 separate lists.
Also the list size is higher than using sublists. This way it's hard to find elements in it and also hard to read it.

If you check my other comment the whole list would look like this: https://github.com/Ericsson/codechecker/pull/2723#discussion_r424227171

By the way this is my opinion, we should ask the others. @gyorb @bruntib What do you think?